core/module_protocols/Mysql.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import copy
+import mysql.connector as sqltor
+
+# from core.utility import reverse_and_regex_condition
+from core.utility import process_conditions
+from core.utility import get_dependent_results_from_database
+from core.utility import replace_dependent_values
+
+
+# def response_conditions_matched(sub_step, response):
```

Comment:
let's clean up unnecessary comments, should do in other modules too.

Reply:
Sure I will do it for this file as well as for other files too.